### ISSUE TITLE
INTERNAL ERROR -- Please try using mypy master on Github:

### ISSUE THREAD

**Issue by jwsloan** [xoeye/typecats#2]
I received this error when trying to run mypy on a very simple playground project using typecats.

```
error: INTERNAL ERROR -- Please try using mypy master on Github:
https://mypy.rtfd.io/en/latest/common_issues.html#using-a-development-mypy-build
If this issue continues with mypy master, please report a bug at https://github.com/python/mypy/issues
```

I tried the development build as suggested, but still had no luck.

@petergaultney, this one's for you. :)

**Comment by petergaultney** [comment]
This was recently added to the [README](https://github.com/xoeye/typecats#mypy-plugin) as a known issue with `mypy >0.740`.

I'm going to leave this issue open to track our progress fixing this for mypy 0.750+.

**Comment by petergaultney** [comment]
This is closed by v1.3.0
